bigtable/client/table.cc

```diff
@@ -107,5 +107,9 @@ void Table::BulkApply(BulkMutation&& mut) {
   }
 }
 
+RowReader Table::ReadRows(RowSet row_set, int rows_limit, Filter filter) {
+  return RowReader(client_, table_name(), row_set, rows_limit, filter,
```

Comment:
You probably want to `std::move()` the "expensive" parameteres (`row_set`, `filter`).

Reply:
done